Actions.js

```diff
@@ -0,0 +1,17 @@
+function myFunction() {
+  const x = document.getElementById('nav-links');
+  if (x.style.display === 'block') {
+    x.style.display = 'none';
+  } else {
+    x.style.display = 'block';
+  }
+}
+
+function changeIcon() {
+  const icon = document.querySelector('i');
+  icon.classList.toggle('fa-bars');
+  icon.classList.toggle('fa-close');
+}
+
+document.getElementById('nav-links').addEventListener('click', myFunction);
+document.getElementById('btn').addEventListener('click', changeIcon());
```

Comment:
- [ ] [REQUIRED] You have done a great job so far @OnlyUnknown 👍🏽 . However, you still need to ensure that the mobile page covers the display screen 100%. I am not supposed to see `p` tag with the classNmae 'content' when I click the mobile menu, I am supposed to see only 'Projects', 'About', and 'Contact'. Kindly fix this before asking for another review.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="300" alt="Screenshot 2023-02-06 at 22 25 25" src="https://user-images.githubusercontent.com/58692590/217094183-5f023316-872f-49d0-b4a9-5480fe0481b1.png"> | <img width="221" alt="Screenshot 2023-02-06 at 22 26 10" src="https://user-images.githubusercontent.com/58692590/217094235-b83f63c6-cc20-48fc-a447-b1d262df272a.png">

Reply:
@OnlyUnknown I can can see that you put a fixed min-width.
<img width="1438" alt="Screenshot 2023-02-06 at 22 34 42" src="https://user-images.githubusercontent.com/58692590/217094543-e7e28344-95af-426b-9274-90a10e58e705.png">